### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added release notes for 4.10

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,48 @@
...
+- `[Field Options]` Fixed mobile support so they now work on touch better on IOS and Android. ([#555](https://github.com/infor-design/enterprise-ng/issues/555))
+- `[Tree]` Tree with + and - for the folders was inversed visually. This was fixed, update your svg.html ([#685](https://github.com/infor-design/enterprise-ng/issues/685))
+- `[Modal]` Fixed an alignment issue with the closing X on the top corner ([#662](https://github.com/infor-design/enterprise-ng/issues/662))
+- `[Popupmenu]` Fixed a visual flickering when opening dynamic submenus. ([#588](https://github.com/infor-design/enterprise/issues/588))
+- `[Tree]` Added full unit and functional tests ([#264](https://github.com/infor-design/enterprise/issues/264))
+- `[Lookup]` Added full unit and functional tests ([#344](https://github.com/infor-design/enterprise/issues/344))
+- `[Datagrid]` Added more unit and functional tests ([#242](https://github.com/infor-design/enterprise/issues/242))
+- `[General]` Updated the develop tools and sample app to Node 10. During this we know ignore package.lock.json in .gitignore ([#540](https://github.com/infor-design/enterprise/issues/540))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]: Should be package-lock.json, with a hyphen.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -5,9 +5,48 @@
...
+- `[Modal]` Allow beforeOpen call back to run optionally whether you have content or not passed back. ([#409](https://github.com/infor-design/enterprise/issues/409))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]: callback is one word.

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -5,9 +5,48 @@
...
+- `[Datagrid]` The lookup editor now supports left, right, center align on the column settings. ([#228](https://github.com/infor-design/enterprise/issues/228))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]: left, right, and center align

**Comment by tmcconechy** [comment]
@davidcarlsonberg Thanks. I made those edits

**Pull request merged**